actor/src/main/scala/org/apache/pekko/actor/LightArrayRevolverScheduler.scala

```diff
@@ -193,6 +195,12 @@ class LightArrayRevolverScheduler(config: Config, log: LoggingAdapter, threadFac
       task
     }
 
+  private def checkPeriod(delay: FiniteDuration): Unit =
```

Comment:
@Roiocam @mdedetrich I think when we are not using @inline and iniline def, we may just online this method？but I think the jvm should do the same after some iterations.